axoproject/src/tests.rs

```diff
@@ -5,6 +5,15 @@ use crate::{
     WorkspaceKind,
 };
 
+fn get_package(packages: &[(PackageIdx, &crate::PackageInfo)], name: &str) -> crate::PackageInfo {
+    packages
+        .iter()
+        .find(|(_, p)| p.name == name)
+        .unwrap()
```

Comment:
are we sure this unwrap is safe? we may want to change it to an expect at the very least

Reply:
I'll swap over to an `expect`. This is in the tests so we want it to crash if it's not there, but failing with more informative info will be more helpful.